### PR TITLE
AP_Scripting: added set_override method for scripting

### PR DESCRIPTION
**Pull request by tridge**
useful for test code

### CODE REVIEW

**Review by IamPete1 — COMMENTED** [review]
Needs docs.

**Comment by tridge** [comment]
> Needs docs.

fixed

File: libraries/AP_Scripting/docs/docs.lua [hunk 1]
```diff
@@ -668,6 +668,10 @@ local RC_Channel_ud = {}
 ---@return number
 function RC_Channel_ud:norm_input_ignore_trim() end
 
+-- desc
+---@param param1 integer -- PWM value
+function RC_Channel_ud:set_override(param1) end
```
Comment (IamPete1) [inline libraries/AP_Scripting/docs/docs.lua:673]:
```suggestion
---@param PWM integer
function RC_Channel_ud:set_override(PWM) end
```
As i mentioned on the other PR. This is what I have been doing.

Comment (tridge) [reply libraries/AP_Scripting/docs/docs.lua]: done

**Review by IamPete1 — APPROVED** [review]

**Pull request merged**